src/main/resources/db/migration/0200/V226__OrganizationTypes.sql

```diff
@@ -0,0 +1,23 @@
+CREATE TABLE managed_facility_types (
+    id INTEGER NOT NULL PRIMARY KEY,
+    name TEXT NOT NULL
+);
+
+CREATE TABLE organization_types (
+    id INTEGER NOT NULL PRIMARY KEY,
+    name TEXT NOT NULL
+);
+
+CREATE TABLE organization_managed_facility_types (
+  organization_id BIGINT NOT NULL REFERENCES organizations,
+  managed_facility_type_id INTEGER NOT NULL REFERENCES managed_facility_types,
+  PRIMARY KEY (organization_id, managed_facility_type_id)
+);
+
+ALTER TABLE organizations ADD COLUMN organization_type_id INTEGER REFERENCES organization_types;
+ALTER TABLE organizations ADD COLUMN organization_type_details VARCHAR(100);
+ALTER TABLE organizations ADD COLUMN website TEXT;
+
+/* Require non-empty type details if type id is 'Other' (6), type details should be null otherwise. */
+ALTER TABLE organizations ADD CONSTRAINT other_type_details
+  CHECK ((organization_type_details IS NULL AND organization_type_id != 6) OR (organization_type_details <> '' AND organization_type_id = 6));
```

Comment:
Sorry to nitpick! But I think this still allows `organization_type_details IS NULL` at the same time as `organization_type_id = 6`

Reply:
Should the second clause be `(organization_type_details IS NOT NULL AND organization_type_details <> '' AND organization_type_id = 6)`?

Reply:
No worries. I believe `<> ''` catches not null and not empty.

Reply:
Fwiw, I am adding tests in an upcoming PR.

Reply:
>No worries. I believe <> '' catches not null and not empty.

Okay! I didn't realize that.

Reply:
> > No worries. I believe <> '' catches not null and not empty.
> 
> Okay! I didn't realize that. On googling a little bit, you seem to be correct 😆

I added the check anyways to avoid confusion :)

Reply:
Ok looks like the `<> ''` only may work in sql but not postgresql. Testing locally in psql I needed both the 'is not null' and '<> ''' conditions.

Reply:
Oh! Thanks so much for testing that out ❤️